core/src/main/scala/precog/SbtPrecogBase.scala

```diff
@@ -205,15 +205,15 @@ abstract class SbtPrecogBase extends AutoPlugin {
       developers := List(
         Developer(
           id = "precog",
-          name = "Precog Inc.",
-          email = "contact@precog.com",
+          name = "Precog Data",
+          email = "support@precog.com",
           url = new URL("http://precog.com")
         )))
 
     lazy val githubActionsSettings: Seq[Def.Setting[_]] = Seq(
       githubWorkflowSbtCommand := s"$$SBT",
 
-      githubWorkflowJavaVersions := Seq("adopt@1.8", "graalvm8@20.1.0"),
+      githubWorkflowJavaVersions := Seq("adopt@1.11", "graalvm-ce-java11@21.1", "graalvm-ce-java8@21.1"),
```

Comment:
Unsure if we want to make this change now or not. I'd like to get on Java 11, if possible, but not if it will take too much time.

The versions I chose here are the ones we're currently running workflows against in `precog/precog`.